Cut(eos_cli_config_gen)!: Remove the logic for counter creation under traffic-policies.j2 (#5851)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

| `management_ssh.access_groups` | `management_ssh.ip_access_group_in or management_ssh.vrfs.ip_access_group_in` |
| `management_ssh.ipv6_access_groups` | `management_ssh.ipv6_access_group_in or management_ssh.vrfs.ipv6_access_group_in` |

> ### All the traffic policies counters should be defined under `traffic_policies.policies[].counters`, it will no longer be added automatically

Reviewer: ```suggestion
### Traffic policy named counters must now be explicitly defined under `traffic_policies.policies[].counters`
```

> In AVD 5.0.0, Counters were generated automatically based on `traffic_policies.policies[].matches[].actions[].count`.

Reviewer: ```suggestion
In AVD 5.0.0, named counters were generated automatically based on the `traffic_policies.policies[].matches[].actions[].count` key.
```

> Starting AVD 6.0.0, all the counters should be defined under the new key `traffic_policies.policies[].counters`.

Reviewer: ```suggestion
Starting AVD 6.0.0, this automatic generation has been removed. All named counters must be defined under the new `traffic_policies.policies[].counters` key.
```

```diff
  traffic_policies:
    policies:
      - name: BLUE-C1-POLICY
+       counters:
+         - DEMO-TRAFFIC
+         - DROP-PACKETS

> ```

Reviewer: Let's also add a one or two matches definition with action count here for clarity.

Author: Added!

+    vrf: default
```

### Traffic policy named counters must now be explicitly defined under `traffic_policies.policies[].counters`

In AVD 5.0.0, named counters were generated automatically based on the `traffic_policies.policies[].matches[].actions[].count` key.

Starting AVD 6.0.0, this automatic generation has been removed. All named counters must be defined under the new `traffic_policies.policies[].counters` key.

```diff
  traffic_policies:
    policies:
      - name: BLUE-C1-POLICY
+       counters:
+         - DEMO-TRAFFIC

> +         - DROP-PACKETS

Reviewer 2: let's only add what is relevant to the match below
```suggestion
```
